functions/exercise2/script.js

```diff
@@ -1 +1,30 @@
 "use strict";
+
+function isPalindrome(phrease) {
+    var isPhreasePalindrome = phrease.toLowerCase().replace(/[^a-zA-Z0-9]+/g, "");
+    var counter = 0;
+    if (isPhreasePalindrome === "") {
+        window.alert("Parameter must contain at least one letter - not use white space");
```

Comment:
Do you have window object in unit test execution environment?
It should not be there.
When it is executed in windows environment, then caller from this env will get exception and can react appropriately to this exception by for example displaying this alert.